### PR TITLE
Moved parser presets into class #1662

### PR DESCRIPTION
**Pull request by joachimmetz**
Moved parser presets into class #1662

### CODE REVIEW

File: plaso/parsers/presets.py [hunk 1]
```diff
@@ -1,46 +1,106 @@
 # -*- coding: utf-8 -*-
-"""The parser preset categories."""
+"""The parser and parser pluging presets."""
```
Comment (Onager) [inline plaso/parsers/presets.py]: plugin

File: plaso/parsers/presets.py [hunk 2]
```diff
@@ -1,46 +1,106 @@
...
+    """Initializes a parser and parser plugin preset definition."""
+    super(ParserPresetDefinition, self).__init__()
+    self.name = None
+    self.parsers = None
+
+
+class ParserPresets(object):
+  """Parser and parser plugin presets."""
+
+  CATEGORIES = {
```
Comment (Onager) [inline plaso/parsers/presets.py]: These don't seem to be categories, these are presets by name, using the new terminology. Unless there's some other factor at work, please  at least rename this, or even better, move directly to using the manager approach as elsewhere, and directly create ParserPresetDefinitions.

File: plaso/parsers/presets.py [hunk 3]
```diff
@@ -1,46 +1,106 @@
...
-        'mac_securityd', 'sqlite/mackeeper_cache', 'macwifi', 'olecf',
-        'czip/oxml', 'plist', 'sqlite/skype', 'syslog', 'utmpx', 'webhist',
-        'zsh_extended_history'],
-    'android': [
-        'android_app_usage', 'filestat', 'chrome_cache',
-        'sqlite/android_calls', 'sqlite/android_sms', 'sqlite/android_webview',
-        'sqlite/android_webviewcache', 'sqlite/chrome_cookies',
-        'sqlite/chrome_8_history', 'sqlite/chrome_27_history', 'sqlite/skype']}
+
+class ParserPresetDefinition(object):
```
Comment (Onager) [inline plaso/parsers/presets.py]: -Definition from the name here, as a preset is the name and list of parsers.

File: plaso/parsers/presets.py (hunk 2 above)
Comment (joachimmetz) [reply plaso/parsers/presets.py]: Keeping this as-is, this class constant will be entirely removed in the next step.

File: plaso/parsers/presets.py (hunk 1 above)
Comment (joachimmetz) [reply plaso/parsers/presets.py]: done

File: plaso/parsers/presets.py (hunk 3 above)
Comment (joachimmetz) [reply plaso/parsers/presets.py]: There will be more attributes in this class in the next steps.

**Comment by joachimmetz** [comment]
@Onager this change needs to be considered in the context of multiple steps also see https://github.com/log2timeline/plaso/pull/1662. You are asking for changes for code that will be affected by the next steps.

**Pull request merged**